app/models/node.rb

```diff
@@ -187,14 +187,14 @@ def self.weekly_tallies(type = 'note', span = 52, time = Time.now)
                                .where(type:    type,
                                       status:  1,
                                       created: time.to_i - week.weeks.to_i..time.to_i - (week - 1).weeks.to_i)
-                               .count
+                               .size
     end
     weeks
   end
 
   def self.contribution_graph_making(type = 'note', start = Time.now - 1.year, fin = Time.now)
     date_hash = {}
-    week = start.to_date.step(fin.to_date, 7).count
+    week = start.to_date.step(fin.to_date, 7).size
```

Comment:
Hi, thanks for working on this :100:  lets leave this as `count`...using size here will return nil

Reply:
Thank you for the help. Changes made in latest commit